content/_assets/javascript/application/canvas-panel.js

```diff
@@ -23,12 +23,17 @@ const annotationData = (input) => {
 const getServiceId = (element) => {
   const canvasPanel = element.querySelector('canvas-panel')
   const imageService = element.querySelector('image-service')
+  const imageSequence = element.querySelector('image-sequence')
```

Comment:
@anderspollack we might make a method to abstract getting the `id` for these different elements, sketch below, though we should do so in a separate pull-request and perhaps when we refactor figures?
```
const serviceElement = element.querySelector('canvas-panel', 'image-sequence', 'image-service')
const service = new Service(serviceElement)
return service.id
```

Reply:
@mphstudios I think it makes sense to do that when we refactor all figures. If we can reasonably refactor figure image elements to use webc this file will be obsolete